code/modules/mob/living/basic/icemoon/wolf/wolf.dm

```diff
@@ -0,0 +1,108 @@
+/datum/pet_command/good_boy/wolf
+	speech_commands = list("good wolf")
+
+/datum/pet_command/follow/wolf
+	// Nordic-themed for a bit of extra flavor
+	speech_commands = list("heel", "follow", "fylgja", "fyl")
+
+/datum/component/riding/creature/wolf
+
+/datum/component/riding/creature/wolf/handle_specials()
+	. = ..()
+	set_riding_offsets(RIDING_OFFSET_ALL, list(TEXT_NORTH = list(1, 9), TEXT_SOUTH = list(1, 9), TEXT_EAST = list(0, 9), TEXT_WEST = list(2, 9)))
+	set_vehicle_dir_layer(SOUTH, ABOVE_MOB_LAYER)
+	set_vehicle_dir_layer(NORTH, OBJ_LAYER)
+	set_vehicle_dir_layer(EAST, OBJ_LAYER)
+	set_vehicle_dir_layer(WEST, OBJ_LAYER)
+
+/mob/living/basic/mining/wolf
+	name = "white wolf"
+	desc = "Pack hunters of the Icemoon wastes. While a mere nuisance individually, they become fearsome foes in larger groups."
+	icon = 'icons/mob/simple/icemoon/icemoon_monsters.dmi'
+	icon_state = "whitewolf"
+	icon_living = "whitewolf"
+	icon_dead = "whitewolf_dead"
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	mouse_opacity = MOUSE_OPACITY_ICON
+	speak_emote = list("howls")
+	friendly_verb_continuous = "howls at"
+	friendly_verb_simple = "howl at"
+
+	butcher_results = list(
+		/obj/item/food/meat/slab = 2,
+		/obj/item/stack/sheet/sinew/wolf = 2,
+		/obj/item/stack/sheet/bone = 2
+	)
+	crusher_loot = /obj/item/crusher_trophy/wolf_ear
+
+	maxHealth = 130
+	health = 130
+	obj_damage = 15
+	melee_damage_lower = 7.5
+	melee_damage_upper = 7.5
+	attack_vis_effect = ATTACK_EFFECT_BITE
+	melee_attack_cooldown = 1.2 SECONDS
+
+	attack_verb_continuous = "bites"
+	attack_verb_simple = "bite"
+	death_message = "snarls its last and perishes."
+
+	attack_sound = 'sound/weapons/bite.ogg'
+	move_force = MOVE_FORCE_WEAK
+	move_resist = MOVE_FORCE_WEAK
+	pull_force = MOVE_FORCE_WEAK
+
+	ai_controller = /datum/ai_controller/basic_controller/wolf
+
+	//can we tame this wolf?
+	var/can_tame = TRUE
```

Comment:
why does this var exist? is it for mapediting?

Reply:
in the event admins wish to spawn a giant pack of roving wolves for events/grief and not have the station try to coochie-coochie-coo them into being someone's death pack, and map editing too, i suppose

Reply:
it's already merged now, but this var isn't useful for admins because by the time they toggle it the component will already have been added 😅